test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -1333,10 +1363,14 @@ internal bool ShouldReuse(string workspacePath, string trainSetBottleneckCachedV
         [InlineData(ImageClassificationTrainer.Architecture.MobilenetV2)]
         [InlineData(ImageClassificationTrainer.Architecture.ResnetV250)]
         [InlineData(ImageClassificationTrainer.Architecture.InceptionV3)]
-        //Skipping test temporarily. This test will be re-enabled once the cause of failures has been determined
-        [Trait("Category", "SkipInCI")]
         public void TensorFlowImageClassification(ImageClassificationTrainer.Architecture arch)
         {
+            if (NotCentOS7FactAttribute.IsCentOS7())
```

Comment:
>IsCentOS7 [](start = 40, length = 9)

as discussed pleas also skip in Ubuntu as this also hangs on Ubuntu like below:

[https://dev.azure.com/dnceng/public/_build/results?buildId=556713&view=logs&j=28859320-f5de-51e0-1fd2-7bea8c11cf7a&t=28348391-228f-5cbf-8fcd-3272a0755d32](https://dev.azure.com/dnceng/public/_build/results?buildId=556713&view=logs&j=28859320-f5de-51e0-1fd2-7bea8c11cf7a&t=28348391-228f-5cbf-8fcd-3272a0755d32)
[https://dev.azure.com/dnceng/public/_build/results?buildId=550567&view=logs&j=5aa5c7df-492a-5eaf-973a-62b7b0f0ee3b&t=ffdbd604-f3e2-5332-cf61-c8dd00799b47](https://dev.azure.com/dnceng/public/_build/results?buildId=550567&view=logs&j=5aa5c7df-492a-5eaf-973a-62b7b0f0ee3b&t=ffdbd604-f3e2-5332-cf61-c8dd00799b47)
[https://dev.azure.com/dnceng/public/_build/results?buildId=549005&view=logs&j=28859320-f5de-51e0-1fd2-7bea8c11cf7a&t=28348391-228f-5cbf-8fcd-3272a0755d32](https://dev.azure.com/dnceng/public/_build/results?buildId=549005&view=logs&j=28859320-f5de-51e0-1fd2-7bea8c11cf7a&t=28348391-228f-5cbf-8fcd-3272a0755d32)